src/App.js

```diff
@@ -6,6 +6,7 @@ import Register from './pages/register/register';
 import Group from './pages/group/group';
 import Swipe from './pages/swipe/swipe';
 import Upload from './pages/upload/upload';
+import Button from './components/button/button';
 
```

Comment:
add ./src/App.js to .gitignore so you dont push in the changes to the App.js files:
Do the following:
1) remove all the changes you made from App.js
2) push the changes from App.js
3) add App.js to gitignore -> now you can edit app.js however you want to
4) make any changes to your files  (If you need to)
5) push your changes and pull request!